templates/create-auto-scaling-group.template

```diff
@@ -0,0 +1,656 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "This template creates a FortiGate auto scaling group and related components.",
+    "Parameters": {
+        "QSS3BucketName": {
+            "Type": "String",
+            "AllowedPattern": "^[0-9a-zA-Z]+([0-9a-zA-Z-]*[0-9a-zA-Z])*$",
+            "ConstraintDescription": "Deployment resource S3 bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-).",
+            "Description": "S3 bucket name for the Deployment package. Bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-)."
```

Comment:
```suggestion
            "Description": "The name of the S3 bucket you created for the deployment package. It can only contain numbers, lowercase letters, uppercase letters, and hyphens (-), but should not start or end with a hyphen (-)."
```